### PR TITLE
CHK-2229: feat generate jwt into notification url

### PR DESCRIPTION
**Pull request by Fernando-Granato**
depend on https://github.com/pagopa/pagopa-ecommerce-commons/pull/163

#### List of Changes
Added logic for generate token into notification url during NPG build request

#### Motivation and Context
The change was made to handle the jwt token on the NPG notification url

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by infantesimone — DISMISSED** [review]

**Comment by infantesimone** [comment]
maybe it's better to finish the PR here, that is, with the goal of using the common lib

File: .devops/code-review-pipelines.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-2229-env-variable-for-payment-method
```
Comment (Fernando-Granato) [inline .devops/code-review-pipelines.yml:16]:
```suggestion
      ref: main
      
```

File: .devops/code-review-pipelines.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ resources:
       type: github
       name: pagopa/pagopa-ecommerce-local
       ref: main
+
```
Comment (Fernando-Granato) [inline .devops/code-review-pipelines.yml:17]:
```suggestion
```

File: pom.xml [hunk 3]
```diff
@@ -229,6 +229,8 @@
 					<connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
 					<scmVersionType>tag</scmVersionType>
 					<scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+					<scmVersionType>branch</scmVersionType>
+					<scmVersion>CHK-2231-jwt-generation</scmVersion>
```
Comment (Fernando-Granato) [inline pom.xml:233]: Restore this

**Review by infantesimone — DISMISSED** [review]

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -21,4 +21,6 @@ sessionUrl.basePath=${SESSION_URL_BASEPATH}
 sessionUrl.outcomeSuffix=${SESSION_URL_OUTCOME_SUFFIX}
 sessionUrl.cancelSuffix=${SESSION_URL_CANCEL_SUFFIX}
 sessionUrl.notificationUrl=${SESSION_URL_NOTIFICATION_URL}
-npg.sessionsTTL=${NPG_SESSIONS_TTL}
\ No newline at end of file
+npg.sessionsTTL=${NPG_SESSIONS_TTL}
```
Comment (pietro-tota) [inline src/main/resources/application.properties:24]: Add those variables to README

**Pull request merged**